minvws/nl-kat-coordination
# Do not fail silently when deleting non-existing objects in octopoes

### Changes

This changes the API behavior to fail when we delete references that do not exist.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3811

### Demo

See the updated integration test!

### QA notes

Please verify that you do not get a 200 back when using the delete-many API in Octopoes on non-existing references!

---

### Code Checklist



---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
